static/sass/_pattern_lists.scss

```diff
@@ -28,16 +29,26 @@
         }
 
         &.is-active {
-          border-right: 3px solid $color-tabs-active-bar;
-          font-weight: $font-weight-bold;
+          background-color: #e6e6e6;
+          border-left: 3px solid $color-tabs-active-bar;
         }
       }
     }
 
     @media (min-width: $breakpoint-small) {
-      border-right: 1px solid $color-mid-light;
       border-top: 0;
       padding: 0;
     }
   }
+
+  .p-list--accent-strip {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+
+    &__item {
+      border-top: 3px solid $color-border-accent-alt;
+      padding: 4px 0 0 0;
+    }
+  }
```

Comment:
I think we could just add a utility class, rather than create a new variant of the list pattern i.e.

```css
.has-accent-alt-border {
  border-top: 3px solid $color-border-accent-alt;
  padding: 4px 0 0 0;
}
```
That class can then be added to standard `.p-list__item` elements.